build/wd_cc_benchmark.bzl

```diff
@@ -21,6 +21,7 @@ def wd_cc_benchmark(
           "@com_google_benchmark//:benchmark_main",
           "//src/workerd/tests:bench-tools"
         ],
+        malloc = "@com_google_tcmalloc//tcmalloc",
```

Comment:
This will not work on macO/WindowsS. I'm also hesitant to heavily optimize benchmarks for performance, in this case enabling tcmalloc will result in more realistic measurements when running allocation-heavy benchmarks, but for the macOS/Windows release binaries this will not matter or may result in overfitting.